Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -3,125 +3,157 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
+
 ''' IMPORTS '''
 import json
 
 from stix2 import Bundle, ExternalReference, Indicator, Vulnerability
+from stix2 import AttackPattern, Campaign, Malware, Infrastructure, IntrusionSet, Report, ThreatActor
+from stix2 import Tool, CourseOfAction
+from typing import Any, Callable
+
+
+def main():
+    SCOs: dict[str, str] = {
+        "file md5": "[file:hashes.md5 ='{}']",
```

Comment:
same for all values in dict (btw, you missed {} in line 21)
```suggestion
        'file md5': 'file:hashes.md5',
```

Reply:
done